torch/sparse/__init__.py

```diff
@@ -91,6 +90,20 @@ def mm(mat1: Tensor, mat2: Tensor) -> Tensor:
     return torch._sparse_mm(mat1, mat2)
 
 
+sampled_addmm = _add_docstr(_sparse.sampled_addmm, r"""
+sparse.sampled_addmm(input, mat1, mat2, *, beta=1., alpha=1., out=None) -> Tensor
+
+Computes alpha*(mat1 @ mat2) * spy(input) + beta*input, where spy(input) is the sparsity pattern matrix of input.
```

Comment:
nit: I'd also define what "sparsity pattern" means here. and that it's constrained to CSR

Reply:
I updated the documentation.